sdk/dotnet/Makefile

```diff
@@ -51,6 +51,7 @@ lint::
 
 dotnet_test::
 	dotnet test
+	stty echo
```

Comment:
i'd add a comment above this explaining why we need this.

Reply:
👍